Kernel/Net/RTL8168NetworkAdapter.cpp

```diff
@@ -389,8 +391,10 @@ void RTL8168NetworkAdapter::configure_phy()
         TODO();
     case ChipVersion::Version14:
         TODO();
-    case ChipVersion::Version15:
-        TODO();
+    case ChipVersion::Version15: {
+        configure_phy_e_2();
```

Comment:
It's pretty weird that this works, linux's configuration for this is pretty different, not much we can do without a specification though :man_shrugging:

Reply:
I know, but it works, so... I guess we can look into the linux driver to see exactly what's going on there. I will leave it as a task for another PR :)